src/rng.h

```diff
@@ -0,0 +1,15 @@
+#ifndef RNG_H
+#define RNG_H
+
+#include <random>
+
+class rng {
+private:
+  std::default_random_engine gen;
+
+public:
+  rng();
+  size_t randomIndex(size_t length);
```

Comment:
Change to `int` while technically, yes, `size_t` is the correct thing to use for length, we are using `int` everywhere else.